netlify.toml

```diff
@@ -14,10 +14,12 @@ HUGO_ENV = "production"
 HUGO_ENABLEGITINFO = "true"
 
 [context.deploy-preview]
+HUGO_ENV = "development"
 command = "make preview-build"
 
 [context.branch-deploy]
-command = "make preview-build"
+HUGO_ENV = "archive"
```

Comment:
It looks like it's working! Thank you :tada:  Is this something one needs to set every time a branch is archived? Is it the only thing? If so we should simply document it

Reply:
Answering myself, it does not look like it, because this is specific for branches. That's good!

Reply:
That's right, it'll transparently unmark the archived versions as production. No extra steps are required unless future updates of Docsy change that again.